include/ensmallen_bits/sdp/primal_dual_impl.hpp

```diff
@@ -279,10 +200,8 @@ typename MatType::elem_type PrimalDualSolver<DeprecatedSDPType>::Optimize(
   return Optimize(sdp, coordinates, ySparse, yDense, z, callbacks...);
 }
 
-template<typename DeprecatedSDPType>
 template<typename SDPType, typename MatType, typename... CallbackTypes>
-typename MatType::elem_type PrimalDualSolver<DeprecatedSDPType>::Optimize(
-    const SDPType& sdp,
+typename MatType::elem_type PrimalDualSolver::Optimize(const SDPType& sdp,
     MatType& coordinates,
```

Comment:
You'll need to align the parameters here

Reply:
Can't align parameters and meet the <80 char requirement.